plugins/workspace-minimap/test/minimap_tests.mocha.js

```diff
@@ -0,0 +1,332 @@
+              this.primaryMetrics, this.minimapMetrics,
+              click.x, click.y);
+
+          assert.deepEqual(converted, [-500, -1250],
+              'Incorrect center click');
+        });
+
+        test('Bottom right click', function() {
+          const click = {
+            x: this.minimapMetrics.svgWidth,
+            y: this.minimapMetrics.svgHeight};
+          const converted = Minimap.minimapToPrimaryCoords(
+              this.primaryMetrics, this.minimapMetrics,
+              click.x, click.y);
+
+          assert.deepEqual(converted, [-1500, -2250],
+              'Incorrect bottom right click');
+        });
+      });
+    });
+
+
+suite('Positioning the minimap in the primary workspace', function() {
+  setup(function() {
+    this.mockMetrics = {
+      viewMetrics: {
+        height: 1000,
+        width: 1000,
+      },
+      absoluteMetrics: {
+        left: 0,
+        top: 0,
+      },
+      toolboxMetrics: {
+        position: Blockly.TOOLBOX_AT_LEFT,
+      },
+    };
+
+    this.options = {
+      RTL: true,
+      horizontalLayout: true,
+      toolboxPosition: 'start',
+      move: {scrollbars: true},
+      scrollbar: {
+        isVisible() {
+          return true;
+        },
+        canScrollVertically() {
+          return true;
+        },
+        canScrollHorizontally() {
+          return true;
+        },
+      },
+      getMetrics() {
+        return {viewWidth: 100};
+      },
+    };
+  });
+
+  test('LTR Vertical Start', function() {
+    this.options.RTL = false;
+    this.options.horizontalLayout = false;
+    this.options.toolboxPosition = 'start';
+    this.mockMetrics.absoluteMetrics = {top: 0, left: 107};
+    this.mockMetrics.toolboxMetrics.position = Blockly.TOOLBOX_AT_LEFT;
+
+    const minimap = new PositionedMinimap(this.options);
+    minimap.position(this.mockMetrics, []);
+    const position = minimap.getBoundingRectangle();
+
+    assert.equal(position.top, 20, 'LTR Vertical Start: Incorrect top');
+    assert.equal(position.left, 872, 'LTR Vertical Start: Incorrect left');
+  });
+
+  test('LTR Vertical End', function() {
+    this.options.RTL = false;
+    this.options.horizontalLayout = false;
+    this.options.toolboxPosition = 'end';
+    this.mockMetrics.absoluteMetrics = {top: 0, left: 0};
+    this.mockMetrics.toolboxMetrics.position = Blockly.TOOLBOX_AT_RIGHT;
+
+    const minimap = new PositionedMinimap(this.options);
+    minimap.position(this.mockMetrics, []);
+    const position = minimap.getBoundingRectangle();
+
+    assert.equal(position.top, 20, 'LTR Vertical End: Incorrect top');
+    assert.equal(position.left, 20, 'LTR Vertical End: Incorrect left');
+  });
+
+  test('LTR Horizontal Start', function() {
+    this.options.RTL = false;
+    this.options.horizontalLayout = true;
+    this.options.toolboxPosition = 'start';
+    this.mockMetrics.absoluteMetrics = {top: 35, left: 0};
+    this.mockMetrics.toolboxMetrics.position = Blockly.TOOLBOX_AT_TOP;
```

Comment:
If the toolbox is at the top, would it make more sense for the minimap to be in the bottom left corner?

Reply:
I could definitely change that. I just wanted to keep it consistent with the backpack plugin.